pkg/cri/sbserver/container_create.go

```diff
@@ -355,11 +366,16 @@ func (c *criService) volumeMounts(containerRootDir string, criMounts []*runtime.
 		}
 		volumeID := util.GenerateID()
 		src := filepath.Join(containerRootDir, "volumes", volumeID)
+		if !filepath.IsAbs(dst) && goruntime.GOOS != "windows" {
+			dst = filepath.Join("/", dst)
```

Comment:
maybe we should use `filepath.Clean("/" + dst)` here to ensure it is dst.

Reply:
From https://pkg.go.dev/path/filepath#Join:
> (...) The result is Cleaned (...)

So I think this is fine. Am I missing something?

Reply:
You are correct.